google_auth_oauthlib/flow.py

```diff
@@ -217,18 +218,14 @@ def authorization_url(self, **kwargs):
                 specify the ``state`` when constructing the :class:`Flow`.
         """
         kwargs.setdefault('access_type', 'offline')
-        if not self.code_verifier:
-            chars = ascii_letters+digits+'-._~'
```

Comment:
Would prefer to still allow some way of auto-generating code_verifier so caller doesn't need to worry about how to generate a secure code_verifier. Passing a static string in for instance might give the caller the impression their implementation is secure when it's really not. I'd hope we could be opportunistic here and just auto-generate but that clearly didn't work so maybe the next best option is to add an argument like autogenerate_code_verifier that defaults to False?

If we leave it as you have it here and don't generate code_verifier then importing SystemRandom() from secrets/random should also be removed.

Reply:
It looks like some users currently create separate Flow instances, so the auto-generation doesn't work correctly for them. 

I'd be fine with having the autogenerate_code_verifier default to false. I'll update this PR with that change.